models/Direction.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Coinbase, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+description: |
+  Direction of the relation (i.e. cross-shard/cross-network sends may reference "back" to an earlier transaction
```

Comment:
nit: I'd add a little more context as to what this exactly is for

Reply:
updated it to the best of my knowledge. not sure if it is sufficient.